REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.6
 StringEncodings
 Missings
+Dates
```

Comment:
This shouldn't be there. The ``Dates`` package is deprecated and all its functionality has been moved into ``Base``. Just remove this line entirely.